src/com/lftechnology/java/training/srijan/basics/generics/GenericsInheritance.java

```diff
@@ -0,0 +1,24 @@
+package com.lftechnology.java.training.srijan.basics.generics;
+
+import java.util.logging.Logger;
+public class GenericsInheritance {
+	private static Logger LOGGER = Logger.getLogger(GenericsInheritance.class.getName());
+	public static void main(String[] args){
+		String str = "Srijan";
+		Object obj = new Object();
+		obj = str;
+		LOGGER.info("obj content: " + obj );
+		
+		MyClass<String> myClass = new MyClass<String>();
+		MyClass<Object> myObject = new MyClass<Object>();
+		obj = myClass;
+		//myObject = myClass;//because myObject and myClass are different
+		LOGGER.info("obj content: " + obj );
+		obj = myObject;
+		LOGGER.info("obj content: " + obj );
+	}
+	
+	public static class MyClass<T>{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2094)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) T is not used in the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2326)